src/gsi/gsi_fedOper.F90

```diff
@@ -0,0 +1,185 @@
+module gsi_fedOper
+!$$$  subprogram documentation block
+!                .      .    .                                       .
+! subprogram:	 module gsi_fedOper
+!   prgmmr:	 j guo <jguo@nasa.gov>
+!      org:	 NASA/GSFC, Global Modeling and Assimilation Office, 610.3
+!     date:	 2018-08-10
+!
```

Comment:
Line 3-8: Is Guo involved in the development of this gsi_fedOper.f90 file? If not, suggest removing these lines.

Reply:
No involved.  I will update the documentation accordingly.